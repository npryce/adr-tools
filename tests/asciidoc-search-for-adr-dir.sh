echo "adoc" > .adr-file-type
adr new First Record
mkdir subdir
cd subdir
adr new Second Record
adr list
cd ..
adr list
