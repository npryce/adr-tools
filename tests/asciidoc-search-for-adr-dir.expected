echo "adoc" > .adr-file-type
adr new First Record
doc/adr/0001-first-record.adoc
mkdir subdir
cd subdir
adr new Second Record
../doc/adr/0002-second-record.adoc
adr list
../doc/adr/0001-first-record.adoc
../doc/adr/0002-second-record.adoc
cd ..
adr list
doc/adr/0001-first-record.adoc
doc/adr/0002-second-record.adoc
